x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -267,11 +267,23 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 
 		// if this returns an error it means this is legacy consumer
 		if !legacy {
-			err = k.projectsKeeper.AddComputeUnitsToProject(ctx, clientAddr.String(), uint64(relay.Epoch), relay.CuSum)
+			err = k.projectsKeeper.ChangeComputeUnitsToProject(ctx, clientAddr.String(), uint64(relay.Epoch), relay.CuSum)
```

Comment:
why is it change CU instead of Add cu? cu sum is the sum of CU per session

Reply:
done

Reply:
Sigh ... @Yarom, in my comment I suggested to use "ChargeComputeUnitsToProject()" (note!  "Charge" not "Change"). The reason is that we charge those CUs, take them away, and the code subtracts them from the remaining CUs of the project (whereas using "Add" is thus misleading).

Reply:
done